web-api/src/persistence/postgres/users/getInternalUsers.ts

```diff
@@ -0,0 +1,19 @@
+import { getDbReader } from '@web-api/database';
+import {
+  ADC_SECTION,
+  DOCKET_SECTION,
+  PETITIONS_SECTION,
+} from '../../../../../shared/src/business/entities/EntityConstants';
+import { DbUser, rawUser } from '@web-api/persistence/postgres/users/mapper';
+
+export const getInternalUsers = async (): Promise<DbUser[]> => {
+  const users = await getDbReader(db =>
+    db
+      .selectFrom('dwUser')
+      .where('section', 'in', [DOCKET_SECTION, PETITIONS_SECTION, ADC_SECTION])
```

Comment:
I believe the "general" role is an "internal" user. And possibly (probably?) chambers as well.